packages/clerk-js/src/utils/queryStateParams.ts

```diff
@@ -0,0 +1,40 @@
+import { getClerkQueryParam } from '../utils';
+
+export const decodeBase64Json = (value: string) => {
```

Comment:
These utilities exist in `encoders.ts`. Please fetch them from there.

Reply:
This doesn't look like its used anywhere except for tests.
- Do we still need it?
- /utils exports an `encodeB64` maybe it also exports the corresponding decode method?
- If not, lets move it there and replace the JSON.parse inside readStateParam

Reply:
@nikosdouvlis I removed the the helper that was not used